### DISCUSSION TITLE
220902 일간회고

### DISCUSSION THREAD

**Discussion by yjjjwww** [codestates/BEB-05-Team-Vote#33] [일간회고]
### 오늘은 어떻게 프로젝트에 기여했나요?
- 타입스크립트와 express에 대하여 공부했고, server 기초 세팅을 깃헙에 올렸습니다.
- supabase 문서를 보며 공부했습니다.

### 오늘의 프로젝트에서 힘든 점은 무엇인가요?

- 아직 서버에서 타입스크립트 사용법을 잘 모르겠습니다.

### 내일은 프로젝트에 기여하기 위해 무엇을 해야 하나요?

- [ ] 다른 타입스크립트 사용한 프로젝트 보고 공부하겠습니다.
- [ ] supabase와 prisma 공부를 하겠습니다.

**Comment by thyoondev** [comment]
타입스크립트에 대해 너무 부담 안가지셔도 돼요! 
혹시 타입스크립트가 아직 헷갈린다면 조금 더 공부하면서 천천히 진행해도 될 것 같아요. 
프로젝트 완성도 중요하지만 결국 프로젝트는 공부를 위한거고, 이해하지 못한채로 사용에만 급급하다보면 두 마리 토끼를 놓칠 수도 있으니까요.
화이팅입니다!!!
